authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,36 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+from rest_framework_jwt.settings import api_settings
+import datetime
+import jwt
+from django.conf import settings
+from rest_framework import exceptions
+from rest_framework.authentication import TokenAuthentication
+secret_key = settings.SECRET_KEY
 
 """Configure JWT Here"""
 
+
+class JWTokens(object):
+    def create_token(self, user):
+        """
+        Create and return the user's JWT token
+
+        """
+        jwt_payload_handler = api_settings.JWT_PAYLOAD_HANDLER
+        jwt_encode_handler = api_settings.JWT_ENCODE_HANDLER
+
+        payload = jwt_payload_handler(user)
+        token = jwt_encode_handler(payload)
+
+        return token
+
+"""Configure JWT Here"""
+
+
+class GetAuthentication(TokenAuthentication):
```

Comment:
Introduce doc strings for every method, function and class

Reply:
On it